gui-wm/tinywl/tinywl-0.18.0.ebuild

```
@@ -0,0 +1,47 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="\"minimum viable product\" Wayland compositor based on wlroots"
+HOMEPAGE="https://gitlab.freedesktop.org/wlroots/wlroots"
```

_Hello @sevz17,_
I suppose it would be better to have here a link to a directory that contains the corresponding code.
```suggestion
HOMEPAGE="https://gitlab.freedesktop.org/wlroots/wlroots/-/tree/master/tinywl"
```